### PR TITLE
Actions on content packages - use table?

### PR DESCRIPTION
**Pull request by hal-ey**
I think using a table may make it easier to follow this step. Since the content package was just created, I assume "Delete" may be the last thing someone wants to do, so I put it at the end. Or is the order like that on the UI (Update, Delete, Deploy)?

### CODE REVIEW

**Comment by AnatKoren** [comment]
Thank you for your input, we will review it.

File: docs/30-ContentPackages/creating-a-content-package-9027b86.md [hunk 1]
```diff
@@ -138,11 +138,13 @@ You’ve created a dev space with the *Development Tools for SAP Build Work Zone
...
 5.  Choose *Finish*.
 
-    You’ve successfully created a project that contains the required content package. The newly created content package is now available in your workspace. You can now perform the following:
-
-    -   Update the content package: For more information, see [Updating a Content Package](updating-a-content-package-de85e4f.md)
-    -   Delete the content package: To do so, navigate to the newly created card project. Right-click on the project and select *Delete*.
-    -   Deploy the content package: For more information, see [Deploying a Content Package](deploying-a-content-package-5556cbf.md)
+    You’ve successfully created a project that contains the required content package. The newly created content package is now available in your workspace. You can now perform the following actions on your content package:
+    
+    | Action for content package |   Documentation |
```
Comment (AnatKoren) [inline docs/30-ContentPackages/creating-a-content-package-9027b86.md:143]: Thanks for table. Do you think that the second header could be More Info?
```suggestion
    | Action for content package |   More Information |
```

**Review by AnatKoren — CHANGES_REQUESTED** [review]
Minor comment.

**Comment by AnatKoren** [comment]
@LindsayBert What do you think?

**Comment by hal-ey** [comment]
https://people.sap.com/jens.haley

**Pull request merged**